internal/loader.go

```diff
@@ -267,27 +267,55 @@ func (tl TypeLoader) LoadSchema(args *ArgType) error {
 		return err
 	}
 
+	// load partitioned tables
+	partitionMap, err := tl.LoadRelkind(args, Partition)
+	if err != nil {
+		return err
+	}
```

Comment:
1. Same as in types.go, partitionMap is measleading. It referrs to partitioned tables, not partitions.

1. If we load partitioned tables before normal tables, we can update EcludeTables automatically, so we don't have to specify --exclude-tables explicitly:

```
for pt := range partitionMap {
    args.ExcludeTables = append(args.ExcludeTables, regexp.QuoteMeta(ptName)+"_.*")
}
```

Reply:
Good point. Done it Works fine. However only if partition name would be `foo_xxx`. 
Is there any possibility that partitions could be named like `xxx_foo`?
However, I think it has no sense to name it like this.